oracledb-ballerina/types.bal

```diff
@@ -0,0 +1,145 @@
+// Copyright (c) 2021 WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+//
+// WSO2 Inc. licenses this file to you under the Apache License,
+// Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+import ballerina/jballerina.java;
+import ballerina/sql;
+
+
+# IntervalYearToMonth stores a period of time in years and months.
+#
+# + years - Number of years
+# + months - Number of months
+type IntervalYearToMonth record {|
+    int years;
+    int months;
+|};
+
+
+# IntervalDayToSecond stores a period of time in days, hours, minutes, and seconds.
+#
+# + days - Number of days
+# + hours - Number of hours
+# + minutes - Number of minutes
+# + seconds - Number of seconds
+type IntervalDayToSecond record {|
+    int days;
+    int hours;
+    int minutes;
+    decimal seconds;
+|};
+
+
+# ObjectType is an abstraction of the real-world entities, such as purchase orders, that application programs deal with.
+#
+# + typename - Name of the object type
+# + attributes - Attributes of the object
+type ObjectType record {|
+    string typename;
```

Comment:
Shouldn't this be `typeName`?

Reply:
It was changed from `typeName` to `typename` considering it as one word, acc to the design review suggestions .